### DISCUSSION TITLE
Change font/font size

### DISCUSSION THREAD

**Discussion by squatto** [spatie/laravel-ray#64] [Ideas]
It would be helpful to be able to change the font and font size. Thanks!

**Comment by freekmurze** [comment]
This is already supported, take a look in the "View" menu.

**Reply by squatto** [reply]
Are you referring to zoom? That works for an overall increase/decrease of the entire app's content, for sure. I should have been more specific in my suggestion: I'd like to change the font and font size of the code output.
